Libraries/Linking/Linking.js

```diff
@@ -139,20 +145,16 @@ class Linking {
    * @platform ios
    */
   static removeEventListener(type: string, handler: Function ) {
```

Comment:
Fixed, but GH is in some sort of weird read-only mode… waiting to see how this pans out…